Source/CarthageKit/Project.swift

```diff
@@ -919,10 +917,10 @@ public final class Project { // swiftlint:disable:this type_body_length
 		let fileManager = FileManager.default
 
 		return self.dependencySet(for: dependency, version: version)
-			.zip(with: // file system objects which might conflict with symlinks
-				list(treeish: version.commitish, atPath: carthageProjectCheckoutsPath, inRepository: repositoryURL)
-					.map { (path: String) in (path as NSString).lastPathComponent }
-					.collect()
+			// file system objects which might conflict with symlinks
+			.zip(with: list(treeish: version.commitish, atPath: carthageProjectCheckoutsPath, inRepository: repositoryURL)
```

Comment:
The linting message I see (*“Colon Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)”*) is mistaken in that this is a method call with an argument label.

We should just use `// swiftlint:disable:previous colon` on the next line to workaround this errant message from the linter.